Packs/Dropbox/ReleaseNotes/1_0_5.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Dropbox Event Collector
+- Updated the Docker image to: *demisto/fastapi:1.0.0.36992*.
+- Documentation and metadata improvements.
```

Comment:
I'm not sure this line is needed.

Reply:
Do you have another suggestion? we can't leave it empty...

Reply:
second line is [standard](https://xsoar.pan.dev/docs/documentation/release-notes#general-changes) when there are no changes, we could either leave it or remove it and only leave the first one, of the docker image update.

Reply:
Leaving just the Docker image update.